### ISSUE TITLE
Crash to Desktop after 1 minute of play time

### ISSUE THREAD

**Issue by VesperMace** [anegostudios/VintageStory-Issues#1367]
**Game Version:** v1.15.7
**Platform:** Windows
**Modded:** No 

### Description
Sudden crash to desktop after I was interacting with something after about a minute of play time, does not occure if I just move around, happeden when putting cooking put down on ground, when clay forming a oven for baking and when placing a crock pot on a shelf (tried diffrent things to see if was something in particular). A clean install of the game did not fix the issue

### How to reproduce
1. Place an object (crock pot is the most frequent) in the world.
3. It crashes to desktop instantly.

### Logs
```
Running on 64 bit Windows with 8 GB RAM 
Version: v1.15.7 (Stable)2021/10/21 12:42:13: Critical error occurred
System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values.
Parameter name: slotId
   at Vintagestory.API.Common.InventoryGeneric.get_Item(Int32 slotId)
   at Vintagestory.GameContent.BlockEntityContainer.OnBlockPlaced(ItemStack byItemStack)
   at Vintagestory.GameContent.BlockEntityCrock.OnBlockPlaced(ItemStack byItemStack)
   at Vintagestory.Client.NoObf.ClientWorldMap.SpawnBlockEntity(String classname, BlockPos position, ItemStack byItemStack)
   at Vintagestory.API.Common.Block.OnBlockPlaced(IWorldAccessor world, BlockPos blockPos, ItemStack byItemStack)
   at Vintagestory.Common.BlockAccessorRelaxed.SetBlock(Int32 blockId, BlockPos pos, ItemStack byItemstack)
   at Vintagestory.API.Common.Block.DoPlaceBlock(IWorldAccessor world, IPlayer byPlayer, BlockSelection blockSel, ItemStack byItemStack)
   at Vintagestory.GameContent.BlockBehaviorHorizontalOrientable.TryPlaceBlock(IWorldAccessor world, IPlayer byPlayer, ItemStack itemstack, BlockSelection blockSel, EnumHandling& handling, String& failureCode)
   at Vintagestory.API.Common.Block.TryPlaceBlock(IWorldAccessor world, IPlayer byPlayer, ItemStack itemstack, BlockSelection blockSel, String& failureCode)
   at Vintagestory.Client.NoObf.ClientMain.OnPlayerTryPlace(BlockSelection blockSelection, String& failureCode)
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.OnBlockBuild(BlockSelection blockSelection, Block onBlock, String& failureCode)
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.HandleMouseInteractionsBlockSelected(Single dt)
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.OnFinalizeFrame(Single dt)
   at Vintagestory.Client.NoObf.ClientEventManager.TriggerRenderStage(EnumRenderStage stage, Single dt)
   at Vintagestory.Client.NoObf.ClientMain.TriggerRenderStage(EnumRenderStage stage, Single dt)
   at Vintagestory.Client.NoObf.ClientMain.RenderToDefaultFramebuffer(Single dt)
   at _gt983m53spH3zIRFvTrVgBj5MUb._T9YfRFmW6X4ZcWzG7vSwet0Rgrs(Single )
   at _2hFm1B1DwOk1Z7yF6DtBdLeBFKaA._5soGXezAcR5oyhhq2pmELCuEn2M(Single )
   at _2hFm1B1DwOk1Z7yF6DtBdLeBFKaA._ZzmaNk5IGbtvQ61ssCzCszyBw2n(Single )
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
   at OpenTK.GameWindow.DispatchRenderFrame()
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   at _LiN8p156auXcqjjOW5aVAZ0QkUc._fwMbTl3XHB2eqbxoLfc3FpMIE00(_CFodeVozKjT5VhVV66Wv4Bd7r3c , String[] )
   at _qBF2AKwdIhCkNZsjxBA24g20vlH._fwMbTl3XHB2eqbxoLfc3FpMIE00(ThreadStart )
-------------------------------

-------------------------------

```

**Comment by tyronx** [comment]
Thanks for the report. I will need more debug info for this which I have added for the next release. Once I have released 1.15.8, please make it crash again and report the crash log here.

**Comment by Craluminum2413** [comment]
Closing due to inactivity
